python/src/KrigingAlgorithm_doc.i.in

```diff
@@ -296,3 +296,22 @@ Returns
 noise : sequence of positive float
     The noise variance :math:`\tau_k^2` of each output value."
 
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::KrigingAlgorithm::getMethod
+"Linear algebra method accessor.
+
+Returns
+-------
+method : str
+       Used linear algebra method."
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::KrigingAlgorithm::setMethod
+"Linear algebra method set accessor.
+
+Parameters
+----------
+method : str
+       Used linear algebra method.
+       Value should be `LAPACK` or `HMAT`."
```

Comment:
A mention of the relevant ResourceMap key would be of interest here

Reply:
ok I will add a Note section